src/dmd/traits.d

```diff
@@ -82,14 +82,16 @@ struct PushAttributes
  * Returns:
  *      Dsymbol  the corresponding symbol for oarg
  */
-private Dsymbol getDsymbolWithoutExpCtx(RootObject oarg)
+Dsymbol getDsymbolWithoutExpCtx(RootObject oarg)
 {
     if (auto e = isExpression(oarg))
     {
         if (e.op == TOK.dotVariable)
             return (cast(DotVarExp)e).var;
         if (e.op == TOK.dotTemplateDeclaration)
             return (cast(DotTemplateExp)e).td;
+        if (e.op == TOK.scope_)
+            return (cast(ScopeExp)e).sds;
```

Comment:
necessary for applying the new syntax to https://github.com/BBasile/iz/blob/master/import/iz/types.d#L482. Unfortunately this custom traits is a bit too complex to copy in the runnable test case (it relies on phobos and on other custom traits)